src/shogun/kernel/GaussianKernel.h

```diff
@@ -189,7 +189,7 @@ class CGaussianKernel: public CDotKernel
 
 	protected:
 		/** width */
-		float64_t width;
+		float64_t m_log_width;
```

Comment:
renaming variables will break integration tests.
Check whether the ones broken are caused by this renaming, it is likely to be the reason